pkg/controller/volume/persistentvolume/pv_controller_base.go

```diff
@@ -126,10 +127,19 @@ func NewController(p ControllerParameters) (*PersistentVolumeController, error)
 	controller.classLister = p.ClassInformer.Lister()
 	controller.classListerSynced = p.ClassInformer.Informer().HasSynced
 	controller.podLister = p.PodInformer.Lister()
+	controller.podIndexer = p.PodInformer.Informer().GetIndexer()
```

Comment:
Do we need to update https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go#L233 to add pod get/list/watch rbacs? I'm surprised we don't see any test failures?

Reply:
It already has these rbacs

https://github.com/kubernetes/kubernetes/blob/0891f69f5ec27cdbcb6336ce3cfe6efdcc1d1e05/plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go#L240